codalab/lib/download_manager.py

```diff
@@ -361,7 +361,8 @@ def _send_netcat_message(self, worker, response_socket_id, uuid, port, message):
     def _get_read_response_stream(self, response_socket_id):
         with closing(self._worker_model.start_listening(response_socket_id)) as sock:
             header_message = self._worker_model.get_json_message(sock, 60)
-            precondition(header_message is not None, 'Unable to reach worker')
+            if header_message is None:
+                logging.info('Unable to reach worker')
```

Comment:
I'm mildly worried that there's an important case that we're suppressing.  Could we have these go to sentry?

Reply:
If we want the error to go to sentry, I think we should leave it as is. Raising `PreconditionViolation` will create a Sentry event